community/examples/hpc-slurm-statesave-persistent.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 Google LLC
```

Comment:
nit. 2024

Reply:
Fixed the copyright as well as added a small description on what this blueprint does